src/redash_stmo/data_sources/query_status/bundle/query_status.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { render } from 'react-dom';
+import { debounce } from 'lodash';
+
+function getQueryStatus($http, query, data_source_id) {
+  return $http.post('api/queries/check', { query, data_source_id });
```

Comment:
I like the use of angular's existing $http here though if the goal is to end its use eventually we'll need to redo these. No sense in pulling in another package just yet though.